onadata/apps/main/urls.py

```diff
@@ -201,11 +201,31 @@
         BriefcaseViewset.as_view({"get": "list", "head": "list"}),
         name="view-submission-list",
     ),
+    re_path(
+        r"^forms/(?P<xform_pk>\w+)/view/submissionList$",
```

Comment:
@KipSigei Should we prefer `submission-list` over `submissuibList`?

Reply:
@kelvin-muchiri The endpoints are specific to the ODK Briefcase app and won't work if they are modified

Reply:
@kelvin-muchiri See https://docs.getodk.org/briefcase-api/